src/skills-builder/skills-builder-modal/skillsBuilderModal.scss

```diff
@@ -1,16 +1,14 @@
-.skills-builder-modal {
-  button[aria-label="Close"][type="button"]{
-    color: $white;
-  }
+button[aria-label="Close"][type="button"]{
+  color: $white;
```

Comment:
Nit: Is there a reason why this selector has been un-nested from the `skills-builder-modal` class selector?

Reply:
Good point. I had a reason at first, and now I realize that the point of wrapping it with the skills builder class selector was so it only affected this component and no other potential modal usages. I'll revert it!